tsqa/conf_plugin.py

```diff
@@ -0,0 +1,28 @@
+import logging
+import os
+
+from nose.plugins import Plugin
+
+log = logging.getLogger('nose.plugins.ConfPlugin')
+
+class ConfPlugin(Plugin):
+
+    name = 'conf-plugin'
+    def options(self, parser, env=os.environ):
+        """Register commandline options.
+        """
+        Plugin.options(self, parser, env)
+        parser.add_option('--keep-tmp', action='store_true', dest='keep_tmp', default=True,
+                          help="Keep tmp files after running successfully")
+        parser.add_option('--sleep-in-sec', type='int', default=0,
+                          dest='sleep_in_sec',
+                          help='Sleep time before ATS start and after ATS stop to allow enough time for async tests')
```

Comment:
I don't think we need this time for "before" tests, since we already wait on the sockets to become available, although this might be useful for waiting on logs to flush to disk after the tests.